### PR TITLE
Add new rules to Checkstyle

### PR DESCRIPTION
**Pull request by JosephineKwa**
Added whitespace after comma rule

### CODE REVIEW

**Comment by wkurniawan07** [comment]
Next time, it's fine to do other unrelated style fixes in the lines you've touched (but no other).

File: src/client/java/teammates/client/scripts/DataGenerator.java [hunk 1]
```diff
@@ -64,7 +64,7 @@
 
     public static void main(String[] args) throws IOException {
         String data = generateData();
-        writeDataToFile(data,TestProperties.TEST_DATA_FOLDER +"/"+ FILE_NAME);
+        writeDataToFile(data, TestProperties.TEST_DATA_FOLDER +"/"+ FILE_NAME);
```
Comment (wkurniawan07) [inline src/client/java/teammates/client/scripts/DataGenerator.java]: e.g this one, feel free to fix the whitespaces around the `+` as well.

Comment (JosephineKwa) [inline src/client/java/teammates/client/scripts/DataGenerator.java]: OK noted!

**Comment by JosephineKwa** [comment]
Sorry forgot to wait for merge to be approved! :persevere:

**Pull request merged**